### ISSUE TITLE
Autosuggest/Autocomplete issues

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/ComETT#132]
Anything but Notes and Dates could benefit from Autocomplete but should be optional for the user (e.g. toggle on/off)

**Comment by BevStubbs** [comment]
I agree but some autocompletes should be limited to the user or file.
